packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -967,6 +973,17 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             metadata,
         );
 
+        if(this.garbageCollector.shouldRunSweep) {
+            const defaultContainerRuntimeExpiryMs = 30 * 24 * 60 * 60 * 1000;
```

Comment:
Declare this const globally and use it here. It makes it easy to find and replace the value.